.github/workflows/publish_package.yaml

```diff
@@ -3,7 +3,7 @@ name: Publish fidesctl
 on:
   push:
     branches:
-      - main
+      - "*"
```

Comment:
This will publish any time we creat or update a breach even before the code is reviewed right? If so is that what we want?

Reply:
@sanders41 good call, I was thinking we want to push a test build to pypi on every commit on every PR but actually that doesn't make much sense, other than here where it was useful for testing?

I think you're right, makes more sense to only do it on a push to main